resources/zib/zib-TreatmentObjective.xml

```diff
@@ -0,0 +1,199 @@
+      <path value="Goal.target.extension" />
+      <sliceName value="medicalDevice" />
+      <min value="0" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-FunctionalOrMentalStatus.MedicalDevice" />
+      </type>
+    </element>
+    <element id="Goal.target.measure">
+      <path value="Goal.target.measure" />
+      <short value="StatusName" />
+      <definition value="Coded description of the functional or mental status or limitation.&#xD;&#xA;Code systems used include:&#xD;&#xA;&#xD;&#xA;SNOMED CT&#xD;&#xA;ICF (International Classification of Functioning, Disability and Health)&#xD;&#xA;NOC (Nursing Outcomes Classification)&#xD;&#xA;OMAHA" />
+      <alias value="StatusNaam" />
+      <binding>
+        <strength value="required" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.121.11.4--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-functionalormentalstatus-v3.2-2020EN" />
+        <map value="NL-CM:4.26.2" />
+        <comment value="StatusName" />
+      </mapping>
+    </element>
+    <element id="Goal.target.detail[x]">
+      <path value="Goal.target.detail[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Goal.target.detail[x]:detailCodeableConcept">
+      <path value="Goal.target.detail[x]" />
+      <sliceName value="detailCodeableConcept" />
+      <short value="StatusValue" />
+      <definition value="The extent to which the patient is limited by the functional or mental condition.&#xD;&#xA;If both the status name and the status value are recorded in code, then both values must come from the same code system. If the patient uses a medical device, the evaluation of the extent of the limitation will take place without this aid. The answer codes must be selected from the subselection of the code system corresponding with the request in accordance with the rules applicable to that system." />
+      <alias value="StatusWaarde" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="required" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.121.11.3--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-functionalormentalstatus-v3.2-2020EN" />
+        <map value="NL-CM:4.26.3" />
+        <comment value="StatusValue" />
+      </mapping>
+    </element>
+    <element id="Goal.statusDate">
+      <path value="Goal.statusDate" />
+      <short value="StatusDate" />
+      <definition value="Date on which the status was determined. A ‘vague’ date, such as only the year, is permitted." />
+      <alias value="StatusDatum" />
+      <mapping>

```

Comment:
I don't think this element can be mapped. It does seem to match with the original intent of FunctionalOrMentalStatus but we are using the 'future' version of the zib within Goal. If you map this to this status field you will combine a future and past recorded of FunctionalOrMentalStatus. Moreover,  on second thought, I also think the semantics of both concepts don't really match. Goal.statusDate is really about the current status of the Goal while that of FunctionalOrMentalStatus would be about the observation OR a desired date in the future for the desired target?

Reply:
So leave out this mapping and explain on the root?

And perhaps also ask the zib centre about this? With regards to TreatmentObject, how should we interpret the FunctionalOrMentalStatus.statusDate?

Reply:
If it concerns a date in the future for the goal, perhaps the `.target.due[x]` would be more fitting.

Reply:
You have changed the lifecycleStatus to only active goals so this will fit

Reply:
Ok, but then you will need to change the definition because no it is still written in the past.

Reply:
I just did it :)